src/conv.jl

```diff
@@ -188,7 +189,17 @@ for (front_name, backend) in (
         function $(Symbol("$(front_name)!"))(
                         out::AbstractArray{T,5}, in1::AbstractArray{T,5},
                         in2::AbstractArray{T,5}, cdims::ConvDims; kwargs...) where {T <: $G}
-            $(Symbol("$(front_name)_$(backend)!"))(out, in1, in2, cdims; kwargs...)
+            x_cs = Iterators.partition(1:size(in1, 4),
+                                       channels_in(cdims) ÷ groupcount(cdims))
+            w_cs = Iterators.partition(1:size(in2, 5),
+                                       channels_out(cdims) ÷ groupcount(cdims))
+            cdims2 = DenseConvDims(cdims, G = 1, C_in = channels_in(cdims) ÷ groupcount(cdims), C_out = channels_out(cdims) ÷ groupcount(cdims))
```

Comment:
@staticfloat  I'm wondering if its acceptable to modify the `cdims`, here. We can't use `DenseConvDims` of course, but I have an overload for `groupcount(::ConvDims)` for the default case anyway.